nixos/modules/virtualisation/kvmgt.nix

```diff
@@ -50,10 +49,7 @@ in {
     };
 
     boot.kernelModules = [ "kvmgt" ];
-
-    boot.extraModprobeConfig = ''
-      options i915 enable_gvt=1
-    '';
+    boot.kernelParams = [ "i915.enable_gvt=1" ];
```

Comment:
> Also moved the kernel option to modprobe configuration - this is somewhat controversial, my reasoning is that options for modules should be there to avoid command line bloat and be a single source of truth.

@abbradar via #64425

Reply:
Ah, sorry, missed your commit message about extraModprobeConfig